File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        int max = Integer.MIN_VALUE;
        for(int i=0;i<candies.length;i++){
            max = Math.max(candies[i],max);
        }
        ArrayList<Boolean> res = new ArrayList<>();
        for(int val : candies){
            res.add((val+extraCandies) >= max);
        }
        return res;
    }
}

File: 1528-kids-with-the-greatest-number-of-candies/1528-kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
    let max = 0;
    for(let cand of candies){
        max = Math.max(max, cand);
    }
    let res = [];
    for(let i=0;i<candies.length;i++){
        if(candies[i] + extraCandies >= max){
            res[i] = true;
        }
        else{
            res[i] = false;
        }
    }
    return res;
};